PharmacyInfo/src/PharmaciesList.vue

```diff
@@ -34,14 +34,14 @@
                         </p>
                     </th>
                     <th scope="col"
-                        class="table-header">
+                        class="table-header" v-if="authRole === 'Pharmacy'">
```

Comment:
Please use enums instead of literals for such cases, because you have the list of roles it may be used not only in one component,
even more, avoid duplication of literals, use place them in a separate file in the folder cnostans/ at the root of the project and use Snake Case (All Caps) for naming, or save it as enums like it's used for roles usualy